README.md

```diff
@@ -16,3 +17,13 @@ Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
+
+Verificiation:
+Choose a variety of input lists, including already sorted lists, reverse sorted lists, lists with duplicate elements, random lists, and lists with special patterns.
+Vary the size of the input lists from small to large to observe the algorithm's behavior across different scales. Evaluate the algorithm's performance on edge cases, such as extremely large input lists, lists with nearly identical elements, or lists with a specific structure that might challenge the sorting process.
```

Comment:
...and what would you expect to observe?

Reply:
The worst case run time would probably be O(n log(n))

Reply:
Why would you expect to observe that?

Reply:
The inherent constraints of comparison-based sorting methods.

Reply:
But what would you expect to observe if the algorithm has the claimed complexity?

Reply:
Even in worst-case scenarios, where the input maximizes the number of comparisons, the algorithm would maintain its linear time complexity. Sorting times would not show a noticeable increase or decrease in worst-case scenarios compared to average or best-case scenarios. The algorithm would demonstrate consistent, linear time performance across different types of data and distributions. Sorting times would not vary significantly between random, sorted, reverse sorted, or special-case inputs.

Reply:
Ok, please describe that. Not sure why you don't expect to see differences between the different cases though.

Reply:
While some variations would be expected, especially in terms of the number of comparisons needed, the algorithm should still maintain its linear time complexity overall. A worst-case scenario might have a higher number of comparisons, it should not significantly deviate from linear scaling in terms of sorting time.

Reply:
Ok, you need to describe all of this in your answer.